Drivers/IWDG_driver/Inc/IndependentWatchdog.h

```diff
@@ -0,0 +1,22 @@
+#ifndef SRC_DRIVERS_INDEPENDENTWATCHDOG_H_
+#define SRC_DRIVERS_INDEPENDENTWATCHDOG_H_
+
```

Comment:
Please fix the filenames according to our style guide: https://uwarg-docs.atlassian.net/wiki/spaces/ZP/pages/2268692486/Style+Standards